plugin/src/main/scala/com/typesafe/genjavadoc/AST.scala

```diff
@@ -85,9 +85,11 @@ trait AST { this: TransformCake ⇒
     def maybeSinceDot = if (since.endsWith(".")) " " else ". "
     def render = s" * @deprecated ${msg}${maybeDot}Since $since${maybeSinceDot}"
 
-    def appendToComment(comment: Seq[String]): Seq[String] = comment match {
-      case Nil => List("/**", render, "*/")
-      case c => c.dropRight(1) ++ List(" *", render, "*/")
+    def appendToComment(comment: Seq[String]): Seq[String] = comment.toList match {
```

Comment:
does performance matter here? then perhaps better to use .toVector that is better for concatenating and last?

Reply:
I don't *suspect* so, but can't hurt to make it a `Vector`